htlcswitch/link.go

```diff
@@ -2286,9 +2286,9 @@ func (l *channelLink) canSendHtlc(policy ForwardingPolicy,
 	// too small for the next hop. If so, then we'll cancel the HTLC
 	// directly.
 	if amt < policy.MinHTLCOut {
-		l.log.Errorf("outgoing htlc(%x) is too small: min_htlc=%v, "+
-			"htlc_value=%v", payHash[:], policy.MinHTLCOut,
```

Comment:
AFAICT, this doesn't actually reverse it, which is a good thing, given this isn't a txid. So this commit is actually unnecessary?

Reply:
I must've been chasing ghosts... I thought I saw reversed payment hashes in the log. But you're of course right, this doesn't change anything at all. Dropping the commit.